store/sqlstore/team_store.go

```diff
@@ -559,6 +559,7 @@ func (s SqlTeamStore) GetAllTeamPageListing(offset int, limit int) ([]*model.Tea
 	return teams, nil
 }
 
+// PermanentDelete deletes from the database the team entry that matches the teamId passed as parameter.
```

Comment:
Maybe we can explain here that this is not a "recuperable" action, and maybe say something like "to soft-delete the team you can Update it with the DeleteAt field set to the current millisecond using `model.GetMillis()`"